docs/configuration/location.rst

```diff
@@ -18,8 +18,12 @@ At any of the listed configuration location, the following file will be read:
 
 - :code:`.PyFunceble.yaml`
     - This file is generated automatically if missing.
-    - This file is may be automatically updated if you choose to merge the
-      upstream configuration when a new key is introduced or removed.
+    - This file is automatically replaced if you choose to merge the
```

Comment:
Ping @ZeroDot1 For review 

Related to: https://github.com/funilrys/PyFunceble/issues/246#issuecomment-883368504